package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amphora",
-  "version": "7.10.0",
+  "version": "7.9.0",
```

Comment:
Do we want to make it version 7.10.1?

Reply:
I think we should scrap the `7.10.x` version.
After merging this the only difference between `7.9.0` and HEAD will be updates to test files.